uxarray/__init__.py

```diff
@@ -1,2 +1,17 @@
 from .grid import *
 from .helpers import *
+
+
+def __version__():
+    # Attempt to import the needed modules
+    try:
+        from importlib.metadata import version as version
+    except ImportError:
+        from importlib_metadata import version as version
+
+    try:
+        __version = version("uxarray")
+        print(__version)
+    except AttributeError:
+        # Incase something happens, such as the library isn't installed
+        __version = "0000"
```

Comment:
If we catch an Error, we can just print out the error here instead of assigning ""0000 to __version

Reply:
I think I'm missing what benefit that would add. Sure, it might be useful to have the stack trace for whatever went wrong, but this code will be run when the package is imported and raising an exception on import just because of a failure to pull the version from metadata wouldn't be productive because there's nothing the end user could or should do about it.

Assigning a dummy value allows the import to succeed, `ux.__version__` to still exist but with a value that indicates there was an issue with pulling the version, and is fairly standard practice anyway.